includes/class-wcmp-be-postcode-fields.php

```diff
@@ -651,20 +651,22 @@ public function save_custom_fields($post_id)
      *
      * @return void
      */
-    public function merge_street_number_suffix($order_id)
+    public function merge_street_number_suffix(string $order_id)
     {
         $order                          = WCX::get_order($order_id);
-        $billingHasCustomAddressFields  = self::isCountryWithSplitAddressFields($_POST['billing_country']);
-        $shippingHasCustomAddressFields = self::isCountryWithSplitAddressFields($_POST['shipping_country']);
+        $shipToDifferentAddress = isset($_POST['ship_to_different_address']);
 
-        if (version_compare(WOOCOMMERCE_VERSION, '2.1', '<=')) {
-            // old versions use 'shiptobilling'
-            $shipToDifferentAddress = !isset($_POST['shiptobilling']);
-        } else {
-            // WC2.1
-            $shipToDifferentAddress = isset($_POST['ship_to_different_address']);
-        }
+        $this->hasBillingCustomAddressFields($order, $shipToDifferentAddress);
+        $this->hasShippingCustomAddressFields($order, $shipToDifferentAddress);
+    }
 
+    /**
+     * @param WC_Order $order
+     * @param bool $shipToDifferentAddress
+     */
+    public function hasBillingCustomAddressFields(WC_Order $order, bool $shipToDifferentAddress): void
+    {
+        $billingHasCustomAddressFields  = self::isCountryWithSplitAddressFields($_POST['billing_country']);
         if ($billingHasCustomAddressFields) {
```

Comment:
ik ben zelf altijd voorstander van dit:
```php
if (!$billingHasCustomAddressFields) {
    return;
}

// de rest
```
in plaats van de hele functie nesten, maar ik zou een pr er ook weer niet op afkeuren